src/pages/commits/ActiveCommits/buildVariantCard/WaterfallTaskStatusIcon.tsx

```diff
@@ -88,37 +88,45 @@ export const WaterfallTaskStatusIcon: React.VFC<
           </TaskStatusWrapper>
         </Link>
       }
-      triggerEvent="hover"
     >
       <div data-cy="waterfall-task-status-icon-tooltip">
-        <TooltipTitle
+        <Body
           data-cy="waterfall-task-status-icon-tooltip-title"
           weight="medium"
         >
           {displayName} {timeTaken && `- ${msToDuration(timeTaken)}`}
-        </TooltipTitle>
+        </Body>
         {loading ? (
           <Skeleton />
         ) : (
-          <>
-            {testResults?.map(({ id, testFile }) => (
-              <TestName key={id}>{testFile}</TestName>
-            ))}
+          <div>
+            <TestList>
+              {testResults?.map(({ id, testFile }) => (
+                <li key={id}>
+                  <TestName>{testFile}</TestName>
+                </li>
```

Comment:
[nit] Optionally, could simplify this slightly:
```suggestion
                <TestName key={id}>{testFile}</TestName>
```

And make TestName an `li` element
```js
const TestName = styled.li`
```

I like the formatting change!

Reply:
great idea! 😃